gateway/api_loader.go

```diff
@@ -119,6 +119,9 @@ func processSpec(spec *APISpec, apisByListen map[string]int,
 		"prefix": "coprocess",
 	})
 
+	if strings.HasPrefix(spec.Proxy.TargetURL, "h2c://") {
+		spec.Proxy.TargetURL = strings.Replace(spec.Proxy.TargetURL, "h2c://", "http://", 1)
+	}
```

Comment:
why are we replacing target scheme here?

Reply:
Because we treat h2c as alias to http. Probably should be followed by conned in source code.

Reply:
I'm confused, from reading `h2c` is not `http` traffic please see this doc comment https://pkg.go.dev/golang.org/x/net/http2/h2c#NewHandler

Reply:
This link actually has good description, saying that h2c is http1.1 compatible ible